grammars/stata.cson

```diff
@@ -71,7 +71,7 @@ patterns: [
         ]
   }
   {
-    match: '\\b(foreach)\\s+((?!in|of).+)\\s+(in|of var(list|lis|li|l)?|of new(list|lis|li|l)?|of num(list|lis|li|l)?)\\b'
+    match: '\\b(foreach)\\s+((?!in |of ).+)\\s+(in|of var(list|lis|li|l)?|of new(list|lis|li|l)?|of num(list|lis|li|l)?)\\b'
```

Comment:
This doesn't make sense because both the `.+` and `\\s+` allow whitespace, so it doesn't look like this change does anything

Reply:
It prevents matching a local macro with a name starting with "in" or "of"
Case in point, I had `foreach indepvar of varlist`...

Reply:
Maybe a `\b` would be more correct -- that was the first solution that popped into my head and worked for me

Reply:
Ok, I think I see. You could probably take out `(?!in|of)` completely, but as it is ` ` would need to be replaced by `\\s+`, otherwise it'll break with two spaces.

Reply:
Agreed. I took it out completely.